decidim-elections/app/models/decidim/elections/trustee.rb

```diff
@@ -10,6 +10,8 @@ class Trustee < ApplicationRecord
       has_many :elections, through: :elections_trustees
       has_many :trustees_participatory_spaces, inverse_of: :trustee, foreign_key: "decidim_elections_trustee_id", dependent: :destroy
 
+      validates :name, uniqueness: true
```

Comment:
Do we need to add this if there is a unique index? I don't see a problem, but I'm just wondering if makes sense to check this in three different places (index, model and form). Maybe with index and form would be enough? What do you think? 🤔

Reply:
Yes, although I think the index and the validation inside the form would be enough, I added it here, so it's more obvious that the `name` attribute is nil by default but if it has a value, this value must be unique. I can remove it, though. 😃